DATA/production/calib/tof-diagn-aggregator.sh

```diff
@@ -5,14 +5,25 @@ source common/setenv.sh
 # ---------------------------------------------------------------------------------------------------------------------
 # Set general arguments
 source common/getCommonArgs.sh
+source common/gen_topo_helper_functions.sh
 ARGS_ALL_CONFIG="keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null;$ALL_EXTRA_CONFIG"
```

Comment:
the line `ARGS_ALL_CONFIG=...` is obsolete and can be removed (see getCommonArgs.sh)

Reply:
done